### DISCUSSION TITLE
请问校园网封禁 frp 怎么办，有其他方法可以进行内网穿透吗

### DISCUSSION THREAD

**Discussion by chunibyo-wly** [natfrp/wiki#467] [使用问题]
请问校园网封禁 frp 怎么办，有其他方法可以进行内网穿透吗

**Comment by ArchGuyWu** [comment]
> 请问校园网封禁 frp 怎么办，有其他方法可以进行内网穿透吗

买国内节点，走国内节点穿透

**Comment by chunibyo-wly** [comment]
你好我想穿透香港学校的服务，之前使用的香港节点，看起来是检测了 frp 流量用一会儿就被封了。向日葵和蒲公英一类的也不让联网。请问有什么建议吗。

**Comment by fakeboboliu** [comment]
您可以参考我们文档的 [相关内容](https://doc.natfrp.com/faq/network.html#zoned-network-restriction)。

但是我们建议您在穿透服务前先咨询相关管理部门，或者自查此类服务是否会造成泄密。

如果您无法联系咨询，或者咨询后要求被拒绝，我们建议您以人身安全为上，不要试图进行这种可能造成学籍不保的行为。
